src/main/java/uk/gov/hmcts/reform/userprofileapi/service/impl/ValidationHelperServiceImpl.java

```diff
@@ -76,4 +81,33 @@ public boolean validateUserPersistedWithException(HttpStatus status) {
         }
         return true;
     }
+
+    @Override
+    public boolean validateUserStatusWithException(UserProfile userProfile, IdamStatus expectedStatus) {
+        if (expectedStatus != userProfile.getStatus()) {
+            auditService.persistAudit(HttpStatus.BAD_REQUEST, ResponseSource.API);
+            final String exceptionMsg = String.format("User is not in %s state", expectedStatus);
```

Comment:
please make it void instead of boolean if you are not adding any logic based on return value.

Reply:
Done.